### PR TITLE
Made the config more consistent and corrected grammar

### PR DESCRIPTION
**Pull request by NamefuI**
I don't know whether you'll like it Falco, but I wanted these changes to be made ever since I started lurking in DarkRP files months ago ;)

### CODE REVIEW

**Comment by fruitwasp** [comment]
Nice dots.

**Comment by NamefuI** [comment]
I know right

File: gamemode/config/config.lua [hunk 1]
```diff
@@ -298,16 +298,16 @@ GM.Config.F1MenuHelpPageTitle = "Wiki page"
 -- Put Steam ID's and ranks in this list, and the players will have that rank when they join.
 GM.Config.DefaultPlayerGroups = {
     ["STEAM_0:0:00000000"] = "superadmin",
-    ["STEAM_0:0:11111111"] = "admin",
+    ["STEAM_0:0:11111111"] = "admin"
```
Comment (FPtje) [inline gamemode/config/config.lua]: Commas after the last entry are done on purpose: people are less likely to make a mistake when they add to the lists.

**Comment by NamefuI** [comment]
Shall I put the commas back in place then?

**Pull request merged**